x-pack/legacy/plugins/maps/public/layers/styles/vector/components/field_select.js

```diff
@@ -58,6 +73,16 @@ export function FieldSelect({ fields, selectedFieldName, onChange, ...rest }) {
     });
   }
 
+  function renderOption(option, searchValue, contentClassName) {
```

Comment:
pull this outside the react-component and pull it up to module scope. It's a static helper. There's no need to keep this inside the component as it doesn't require anything from the closure above.